packages/react/src/components/Charts/BarChart/index.tsx

```diff
@@ -196,11 +203,7 @@ const _BarChart = <K extends ChartConfig>(
                 ? "stack"
                 : undefined
             }
-            fill={
-              highlightLastBar
-                ? (((data: { fill: string }) => data.fill) as unknown as string)
-                : (dataConfig[key].color ?? autoColor(index))
-            }
+            fill={dataConfig[key].color ?? autoColor(index)}
```

Comment:
The type for this is `string | undefined`, I'm not sure why we were passing a function and casting it as string 🤔